src/components/RegistrationSuccessPage/RegistrationSuccessPage.app.js

```diff
@@ -0,0 +1,21 @@
+// App.js
+
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
We don't use this library. Next.js has build in routing system, please see doc's or see how Unda create page